BaseTools/Source/C/DevicePath/GNUmakefile

```diff
@@ -30,6 +30,10 @@ ifeq ($(CYGWIN), CYGWIN)
 endif
 
 ifeq ($(LINUX), Linux)
+ifndef CROSS_LIB_UUID

```

Comment:
which tool defines CROSS_LIB_UUID? Does gcc compiler define it?

Reply:
@lgao4 thanks for catching that. It was meant to be declared in the python script: see example here: https://github.com/microsoft/mu_basecore/blob/d9c18c384889d0c445865f96ac660f6abee6115f/BaseTools/Edk2ToolsBuild.py#L263

This is done because the in-box uuid library will not work for cross compilation so that we need to pull down uuid source and build from scratch. But please let me know if you prefer other methods in doing so, otherwise I will include the python script change in the next push.

Reply:
@bcran , how do you resolve this problem? Do you also pull UUID source code and compile it? I don't want user to compile UUID source. Could we upload the pre-built UUID lib file?

Reply:
@lgao4 I'm not sure we want to support cross-building BaseTools - do we support that on Linux?
Why not just build BaseTools on an ARM version of Windows?

Reply:
This is because there is no pipeline support of Windows on ARM :(